install/mysql/glpi-empty.sql

```diff
@@ -2711,6 +2711,7 @@ CREATE TABLE `glpi_entities` (
   `autofill_decommission_date` varchar(255) NOT NULL DEFAULT '-2',
   `suppliers_as_private` int NOT NULL DEFAULT '-2',
   `anonymize_support_agents` int NOT NULL DEFAULT '-2',
+  `default_contracts_id` int NOT NULL DEFAULT '-2',
```

Comment:
To be compatible with foreign keys manipulation methods, it should be named `contracts_id_default`.